frontend/src/views/web/challenge/manage.html

```diff
@@ -9,6 +9,14 @@
             </div>
         </div>
     </div>
+
+    <div class="row" ng-if="challenge.isChallengeHost && !challenge.approved_by_admin">
```

Comment:
Can you share a screenshot of this?
I would like to discuss the placement of this button.

Reply:
@AyushR1 Can you place the button below "Toggle Participation"?